src/ts/uimanager.ts

```diff
@@ -233,6 +233,11 @@ export class UIManager {
       this.uiVariants = <UIVariant[]>playerUiOrUiVariants;
     }
 
+    // Switch on speed selector by default
+    if (config.playbackSpeedSelectionEnabled === undefined) {
+      config.playbackSpeedSelectionEnabled = true;
+    }
+
     this.player = player;
     this.config = {
```

Comment:
Would be nicer to write

```ts
this.config = {
   ...{
    // Default values
    playbackSpeedSelectionEnabled: true,
  },
  ...config,
  etc...
```

instead of the `undefined` check above.

Reply:
You can also move `autoUiVariantResolve` up here.